src/bootstrap.php

```diff
@@ -1116,7 +1116,7 @@ function (Accept $type, string $id) use ($app) {
         'items' => [],
     ];
 
-    $covers = array_slice(array_reverse($covers), 0, 3);
+    $covers = array_slice(array_reverse($covers), 0, 4);
```

Comment:
I think this change is enough. I don't think we need to add a new cover unless the new cover adds any additional value?